packages/client/hmi-client/src/components/widgets/Slider.vue

```diff
@@ -1,9 +1,9 @@
 <template>
 	<aside
 		:class="`slider ${isOpen ? 'open' : 'closed'} ${direction}`"
-		:style="{ width: isOpen ? contentWidth : tabWidth }"
+		:style="`width: ${isOpen ? contentWidth : tabWidth}`"
```

Comment:
What's the difference here?

Reply:
No worries, it doesn't matter at the end. I was curious if it make things faster/better.